src/components/Copyright/Copyright.tsx

```diff
@@ -159,6 +159,17 @@ const License: React.FC = () => {
         , you may copy, distribute and modify the work as long as any
         modifications are also made available under the GPL.
       </p>
+      <Heading level="h2">In Copyright</Heading>
+      <p>
+        Works that are{" "}
+        <Link to="https://rightsstatements.org/page/InC/1.0/?language=en">
+          In Copyright
+        </Link>{" "}
+        are protected by copyright and/or related rights. You are free to use
+        this Item in any way that is permitted by the copyright and related
+        rights legislation that applies to your use. For other uses you need to
+        obtain permission from the rights-holder(s).
+      </p>
```

Comment:
I think this is ok to leave as is but it requires us to change this language every time we want to update the Copyright page. Is there a more robust test that we can write that is more agnostic to the content of the page?